.github/actions/spell-check/expect.txt

```diff
@@ -19,6 +22,7 @@ ADDUNDORECORD
 ADifferent
 adml
 admx
+Advaith
```

Comment:
this should be moved the the user area in https://github.com/microsoft/PowerToys/blob/e4b59909054399c1cd6b08322cfcc4634f7d8158/.github/actions/spell-check/allow/names.txt#L30